cdc/puller/puller.go

```diff
@@ -39,7 +39,7 @@ const (
 	defaultPullerOutputChanSize = 128000
 )
 
-// Puller pull data from tikv and push changes into a buffer
+// Puller pull data from tikv and push changes into a buffer.
```

Comment:
Please avoid unnecessary change.

```suggestion
// Puller pull data from tikv and push changes into a buffer
```